### ISSUE TITLE
build gets stuck at sd-setup-launcher

### ISSUE THREAD

**Issue by arunpnetskope** [screwdriver-cd/screwdriver#1290]
We have declared a simple job - screwdriver.yaml after updating api image with v0.5.451, it is getting stuck at first step (sd-setup-launcher as per the snapshot attached).

Unfortunately we couldn't find anything in logs except this message:

```
180912/201254.188, [response,api,builds] http://172.16.194.243:32002: get /v4/builds/11 {} 200 (17ms)
180912/201256.631, [response] http://172.16.194.243:32002: options /v4/builds/11/steps/sd-setup-launcher/logs {"from":"0","pages":"10"} 200 (3ms)
180912/201256.868, [response,api,builds,steps,log] http://172.16.194.243:32002: get /v4/builds/11/steps/sd-setup-launcher/logs {"from":0,"pages":10,"sort":"ascending"} 200 (12ms)
180912/201259.143, [response,api,builds] http://172.16.194.243:32002: get /v4/builds/11 {} 200 (15ms)

```

<img width="1411" alt="screen shot 2018-09-12 at 11 07 54 pm" src="https://user-images.githubusercontent.com/39948661/45451401-98699100-b6f8-11e8-8851-715bd248a074.png">

Could we get any assistance in resolving this !

**Comment by arunpnetskope** [comment]
fyi.. it creates a launcher pod which throws an error:

```
root@bskubernetesmaster:~# kubectl logs -f 14
mkfifo: cannot create fifo '/opt/sd/emitter': File exists
[WARN  tini (8)] Tini is not running as PID 1 and isn't registered as a child subreaper.
Zombie processes will not be re-parented to Tini, so zombie reaping won't work.
To fix the problem, use the -s option or set the environment variable TINI_SUBREAPER to register Tini as a child subreaper, or run Tini as PID 1.
/bin/sh: 1: /opt/sd/run.sh: not found

```

**Comment by jithin1987** [comment]
Looks like your launcher version is old. Are you running with latest launcher ? Please re-open if this is still an issue after updating launcher.

**Comment by arunpnetskope** [comment]
I have passed that issue.. not it is getting  stuck at "sd-setup-scm" step and the repo cloning is timing out. I have passed SCM_USERNAME and SCM_ACCESS_TOKEN but doesn't make any difference.

also I have defined secrets with key= SCM_ACCESS_TOKEN and value=itspassword and enabled allowinPR by ticking on checkbox. but pipeline doesn't seem to take these secrets. 

<img width="1167" alt="screen shot 2018-09-18 at 2 53 09 pm" src="https://user-images.githubusercontent.com/39948661/45678044-3e7f2600-bb53-11e8-8cda-f33e960389cd.png">
